lib/publiccloud/azure.pm

```diff
@@ -604,7 +604,7 @@ sub cleanup {
     my ($self, $args) = @_;
     select_host_console(force => 1);
 
-    $self->get_image_version();
+    $self->get_image_version() unless (get_var('PUBLIC_CLOUD_SLES4SAP'));
```

Comment:
maybe in place of
$self->get_image_version() unless (get_var('PUBLIC_CLOUD_SLES4SAP'));
you can do
$self->get_image_version() unless (get_var('PUBLIC_CLOUD_BUILD'));